src/server/qgsserverrequest.cpp

```diff
@@ -20,6 +20,17 @@
 #include "qgsserverrequest.h"
 #include <QUrlQuery>
 
+
+const QString QgsServerRequest::HOST = QStringLiteral( "Host" );
+const QString QgsServerRequest::FORWARDED = QStringLiteral( "Forwarded" );
```

Comment:
Shouldn't this be `X-Forwarded-For` instead?

Reply:
No, this one is the real standard Forwarded header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded

Reply:
Then you need to add `X-Forwarded-For` which to my understanding and experience is the most commonly used header for proxies, especially because you are supporting the other `X-Forwarded-*` non standard headers.

Reply:
And what should I do with 'X-Forwarded-For'?
Just add it?